lib/iris/tests/idiff.py

```diff
@@ -220,7 +220,7 @@ def step_over_diffs(result_dir, action, display=True):
         count = len(results)
 
         for count_index, result_fname in enumerate(results):
-            key = os.path.splitext("-".join(result_fname.split("-")[1:]))[0]
+            key = os.path.splitext("-".join(result_fname.split("result-")[1:]))[0]
```

Comment:
Sorry I didn't pick this up before - CI testing has picked up a formatting issue with `idiff.py`. Please take a look at [Code formatting](https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_code_formatting.html) for how to fix this (and prevent the same in future).

Reply:
Got it, thanks